serpentTools/parsers/detector.py

```diff
@@ -30,8 +31,9 @@ def __init__(self, filePath):
             self._loadAll = True
         else:
             self._loadAll = False
+        self.detCount = 0 # how many dets in file?

```

Comment:
Similar to above comment about the material count, this doesn't need to be an instance attribute

Reply:
done!